responsibleai/responsibleai/_managers/causal_manager.py

```diff
@@ -367,8 +218,44 @@ def name(self):
         return ManagerNames.CAUSAL
 
     def _save(self, path):
-        pass
+        """Save the CausalManager to the given path.
 
-    @staticmethod
-    def _load(path, model_analysis):
-        pass
+        :param path: The directory path to save the ExplainerManager to.
+        :type path: str
+        """
+        causal_dir = Path(path)
+        causal_dir.mkdir(parents=True, exist_ok=True)
+
+        # Save results to disk
+        results_path = causal_dir / SerializationAttributes.RESULTS
+        results_path.mkdir(parents=True, exist_ok=True)
+        for result in self._results:
+            result_path = results_path / result.id
+            result.save(result_path)
+
+    @classmethod
+    def _load(cls, path, model_analysis):
+        """Load the CausalManager from the given path.
+
+        :param path: The directory path to load the ErrorAnalysisManager from.
+        :type path: str
+        :param model_analysis: The loaded parent ModelAnalysis.
+        :type model_analysis: ModelAnalysis
+        """
+        this = cls.__new__(cls)
+        causal_dir = Path(path)
+
+        # Rehydrate results
+        results_path = causal_dir / SerializationAttributes.RESULTS
+        paths = results_path.resolve().glob('*')
+        this.__dict__['_results'] = [CausalResult.load(p) for p in paths]
```

Comment:
should we have load function in causal result to return dict?
So both save and load stay in same file/place for better maintanace in future?